### PR TITLE
Add LoRa Basics Station guide

### LINKED ISSUE #2594: Add Basic Station connection guide
#### Summary

Add a guide to connect Basic Station gateways to TTS. References #1140 

#### Why do we need this ?

Many vendors now support Basic Station. Having a common guide for them with separate sub-guides for vendor specific things makes it easier to use them.

#### What is already there? What do you see now?

No Doc on Basic Station

#### What is missing? What do you want to see?

Document how to connect to TTS CUPS/LNS.

#### How do you propose to document this?

Create a doc and add some examples.

#### Can you do this yourself and submit a Pull Request?

Yes

### PR DESCRIPTION
**Pull request by benolayinka**
#### Summary

Add information about connecting gateways using LBS

Closes #2594 

#### Changes

- LoRa Basics Station section

#### Testing

Run locally

##### Regressions

...

#### Notes for Reviewers

Alternate certificate links:
https://letsencrypt.org/certs/lets-encrypt-x3-cross-signed.pem.txt
https://www.amazontrust.com/repository/AmazonRootCA1.pem

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md`. The target branch is set to `master` if the changes are fully compatible with existing API, database, configuration and CLI.
- [x] Documentation: Relevant documentation is added or updated.
- [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: doc/content/gateways/lora-basics-station/_index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: "LoRa Basics™ Station"
```
Comment (KrishnaIyer) [inline doc/content/gateways/lora-basics-station/_index.md:2]: Isn't it LoRa(TM) @laurensslats?

File: doc/content/gateways/lora-basics-station/_index.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+description: ""
+weight: -1
+---
+
+The [LoRa Basics™ Station](https://lora-developers.semtech.com/resources/tools/basic-station/welcome-basic-station/) protocol simplifies management of large scale LoRaWAN networks. LoRa Basics Station contains two sub protocols for connecting Gateways to Network Servers, LoRa Network Server (LNS) and Configuration and Update Server (CUPS).
```
Comment (KrishnaIyer) [inline doc/content/gateways/lora-basics-station/_index.md:7]: Please define a short code for LoRa™ Basics Station.

File: doc/content/gateways/lora-basics-station/cups.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+---
+title: "Configuration and Update Server Protocol"
+description: ""
+weight: -1
+---
+
+This page contains information about connecting your gateway to {{% tts %}} using the Configuration and Update Server (CUPS) protocol.
```
Comment (KrishnaIyer) [inline doc/content/gateways/lora-basics-station/cups.md:7]: `connecting your gateway to {{% tts %}}` -> `managing your gateway using the  {{% tts %}} CUPS`

File: doc/content/gateways/lora-basics-station/cups.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+1. User account on {{% tts %}} with rights to create gateways.
+
+## Create a Gateway
+
+Follow instructions for [Adding Gateways in the Console]({{< ref "/getting-started/console/create-gateway" >}}) or [Adding Gateways Using the Command-line interface]({{< ref "/getting-started/cli/create-gateway" >}}).
+
+## Create an API Key
+
+CUPS requires an API Key with the following rights:
```
Comment (KrishnaIyer) [inline doc/content/gateways/lora-basics-station/cups.md:21]: Well technically it's not required but we highly recommend having this.

File: doc/content/gateways/lora-basics-station/cups.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+- Edit basic gateway settings
+
+If you have not already created one, follow instructions for [Creating a Gateway API Key in the Console]({{< ref "/getting-started/console/create-gateway#create-gateway-api-key" >}}) or [Creating a Gateway API Key Using the Command-line interface]({{< ref "/getting-started/cli/create-gateway#create-gateway-api-key" >}}).
+
+## Configure Gateway
+
+In your gateway configuration, set the following fields:
+
+CUPS URI: `https://<server-address>:443`
+CUPS Trust: Amazon Root CA 1
```
Comment (KrishnaIyer) [inline doc/content/gateways/lora-basics-station/cups.md:32]: This is only for TTI CH and only currently. A better way is to run 
```
openssl s_client -connect <server-address>:443 -showcerts
```
And use the certs that are provided there.

File: doc/content/gateways/lora-basics-station/lns.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+LNS does not requires an API Key. If you wish to use Token Authentication, create an API Key with the following rights:
+- Link as Gateway to a Gateway Server for traffic exchange, i.e. write uplink and read downlink
+
+If you have not already created one, follow instructions for [Creating a Gateway API Key in the Console]({{< ref "/getting-started/console/create-gateway#create-gateway-api-key" >}}) or [Creating a Gateway API Key Using the Command-line interface]({{< ref "/getting-started/cli/create-gateway#create-gateway-api-key" >}}).
+
+## Configure Gateway
+
+In your gateway configuration, set the following fields:
+
+TC URI: `wss://<server-address>:8887`
```
Comment (KrishnaIyer) [inline doc/content/gateways/lora-basics-station/lns.md:30]: Same as with CUPS, we cannot define the CA for servers that we don't know of.

File: doc/content/gateways/lora-basics-station/_index.md (hunk 1 above)
Comment (benolayinka) [reply doc/content/gateways/lora-basics-station/_index.md:2]: https://github.com/lorabasics/basicstation

File: doc/content/gateways/lora-basics-station/cups.md [hunk 7]
```diff
@@ -18,7 +18,7 @@ Follow instructions for [Adding Gateways in the Console]({{< ref "/getting-start
 
 ## Create an API Key
 
-CUPS requires an API Key with the following rights:
+CUPS does not require an API Key, but it is highly recommended to use one. If you wish to use Token Authentication, create an API Key with the following rights:
```
Comment (KrishnaIyer) [inline doc/content/gateways/lora-basics-station/cups.md:21]: I'm so sorry. The CUPS protocol does not require an API Key but the TTS implementation does. Could you please revert this?

File: doc/content/gateways/lora-basics-station/cups.md [hunk 8]
```diff
@@ -29,6 +29,7 @@ If you have not already created one, follow instructions for [Creating a Gateway
 In your gateway configuration, set the following fields:
 
 CUPS URI: `https://<server-address>:443`
-CUPS Trust: Amazon Root CA 1
+
 CUPS Key: `<gateway-api-key>`
-You can download the CUPS Trust from [here](https://ssl-tools.net/subjects/4f59a39453cfb9559e7f6bd8c54da53da642b714).
+
+CUPS Trust: Use the CA certificate of your trust provider
```
Comment (KrishnaIyer) [inline doc/content/gateways/lora-basics-station/cups.md:35]: Can you create an issue for CH certificates and assign it to me?

Comment (benolayinka) [reply doc/content/gateways/lora-basics-station/cups.md:35]: #2712

**Pull request merged**